src/client/Microsoft.Identity.Client/PlatformsCommon/PlatformNotSupported/ApiConfig/SystemWebViewOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    public partial class SystemWebViewOptions
+    {
+        /// <summary>
+        /// Use Microsoft Edge to navigate to the given URI. On non-windows platforms it uses 
+        /// whatever browser is the default.
+        /// </summary>
+        public static async Task OpenWithEdgeBrowserAsync(Uri uri)

```

Comment:
Please note that there are some warnings coming out of these: 

#[warning]src\client\Microsoft.Identity.Client\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs(15,34): Warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

Please supress these

Reply:
(or make methods non-async?)

Reply:
Good catch. Suppressed warnings.